src/components/textarea/TextArea.js

```diff
@@ -303,7 +303,8 @@ export default class TextAreaComponent extends TextFieldComponent {
         return;
       }
       else {
-        return super.setValue(this.setConvertedValue(value), flags);
+        value = this.component.multiple ? this.getValue() : this.setConvertedValue(value);
```

Comment:
I would think this would still need to set the converted value if multiple... the way you check this is change the editor to save as JSON and then make it multiple. I bet that still breaks here...